### PR TITLE
Common components/prev next button

### PR DESCRIPTION
**Pull request by SajaLahaleeh**

### CODE REVIEW

**Review by sara219 — APPROVED** [review]

File: src/CommonComponents/NextPrevButton/index.js [hunk 1]
```diff
@@ -1,22 +1,30 @@
...
-const NextPrevButton = ({ prevLink, nextLink }) => {
-  return (
-    <Router>
-      <Link to={prevLink}>
-        <PrevButton alt='PreviousButton' src={prev} />
-        <PrevWord>Prev</PrevWord>
+const NextPrevButton = ({ prevLink, nextLink }) => (
+  <Router>
+    <Container>
+      <Link to={prevLink} style={{ textDecoration: 'none' }}>
```
Comment (95Rawan) [inline src/CommonComponents/NextPrevButton/index.js]: Change inline style to styled component

File: src/CommonComponents/NextPrevButton/index.js [hunk 2]
```diff
@@ -1,22 +1,30 @@
...
+        <PrevNextItem>
+          <PrevNextButton alt="PreviousButton" src={prev} />
+          <PrevNextText>Prev</PrevNextText>
+        </PrevNextItem>
       </Link>
-      <Link to={nextLink}>
-        <NextWord>Next</NextWord>
-        <NextButton alt='NextButton' src={next} />
+      <Link to={nextLink} style={{ textDecoration: 'none' }}>
```
Comment (95Rawan) [inline src/CommonComponents/NextPrevButton/index.js]: Change it here too

File: src/CommonComponents/NextPrevButton/index.style.js [hunk 3]
```diff
@@ -1,36 +1,29 @@
...
   font-size: 3.5rem;
   color: var(--next-prev-button-color);
-`
+`;
 
-// export const ToLink = styled.Link`
-// `
+// export const LinkButton = styled.Link`
+//   textdecoration: 'none';
+// `;
```
Comment (95Rawan) [inline src/CommonComponents/NextPrevButton/index.style.js]: Remove comments

File: src/CommonComponents/NextPrevButton/index.js (hunk 1 above)
Comment (SajaLahaleeh) [reply src/CommonComponents/NextPrevButton/index.js]: I did it

**Review by 95Rawan — APPROVED** [review]

**Review by hshahwan — APPROVED** [review]

**Pull request merged**